dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/due-report-initialised/due-report-initialised.event-handler.ts

```diff
@@ -19,7 +20,16 @@ export type UtilisationReportDueReportInitialisedEvent = BaseUtilisationReportEv
  * @param param - The payload
  * @returns The modified report
  */
-export const handleUtilisationReportDueReportInitialisedEvent = (payload: DueReportInitialisedPayload): Promise<UtilisationReportEntity> => {
-  console.error('Utilisation due report error payload %o', payload);
-  throw new NotImplementedError('TODO FN-1860');
+export const handleUtilisationReportDueReportInitialisedEvent = async ({
```

Comment:
Should this be under `try-catch` block?

Reply:
That isn't the pattern we have for the majority of our event handlers - there is handling in the controller that will catch any errors, log them and then return correct response.